experimental/generation/generator/packages/library/templates/standard/library.en-us.lg

```diff
@@ -12,7 +12,7 @@
 - ${template(`${property}_getPropertyNameText`)}
 
 # getPropertyValueText(property, val)
-- ${template(`${property}_getValueText`, val)}
+- ${replace(replace(string(template(`${property}_getValueText`, val)), '\r\n', ''), '"', '\'')}
 
```

Comment:
This seems like a hack.  Shouldn't the underlying template be fixed?

Reply:
This is kind of hacking for the reason that the timex of datetime entity is an array. The return value of ${template(`${property}_getValueText`, val)} has extra '\r\n' and could not be recognized as the correct json.